<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_ui_task</class>
 <widget class="QStackedWidget" name="user_ui_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page_r">
   <widget class="QPushButton" name="pushButton_derive_r">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>获取原文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify_r">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>修改原文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_r">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看译文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sure_r">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>确认完成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_signup_r">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>负责人报名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delay_r">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>推迟ddl</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_subtitle_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>任务简介</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>翻译者</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_tran">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_signu_r_re">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>审核者报名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>删除该任务</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_c">
   <widget class="QPushButton" name="pushButton_c_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查看译文并评价</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c_derive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>获取原文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c_submit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c_allco">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>分配任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_singnup_c">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查看报名者</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delay_c">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>推迟ddl</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_subtitle_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>任务简介</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>翻译者</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_tran_c">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>更新任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>查看发布者评价</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page">
   <widget class="QPushButton" name="btn_look_my_translation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>我的译文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_o_t">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看原文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_t_tr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>继续翻译</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_t_submit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>提交译文</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_subtitle_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>任务简介</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看负责人评价</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看审核人评价</string>
    </property>
   </widget>
   <zorder>pushButton_t_tr</zorder>
   <zorder>btn_look_my_translation</zorder>
   <zorder>pushButton_view_o_t</zorder>
   <zorder>pushButton_t_submit</zorder>
   <zorder>textBrowser_subtitle_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QWidget" name="page_re">
   <widget class="QPushButton" name="pushButton_view_o_t_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看原文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_look_my_translation_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>审核译文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_re_exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看评价</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_subtitle_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>任务简介</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
